deps/toolchains.MODULE.bazel

```diff
@@ -53,3 +53,6 @@ toolchains_musl.config(
     extra_target_compatible_with = ["@com_github_buildbuddy_io_buildbuddy//toolchains:musl_on"],
 )
 use_repo(toolchains_musl, "musl_toolchains_hub")
+
+protoc = use_extension("@toolchains_protoc//protoc:extensions.bzl", "protoc")
+protoc.toolchain(version = "v32.1")
```

Comment:
Could you move this closer to the `protobuf` `bazel_dep` and perhaps even extract the version out into a constant?